tests/foreman/api/test_repositories.py

```diff
@@ -291,3 +291,44 @@ def test_positive_sync_mulitple_large_repos(module_target_sat, module_entitlemen
         data={'ids': [rh_products.id]}, timeout=2000
     )
     assert res['result'] == 'success'
+
+
+def test_positive_available_repositories_endpoint(module_sca_manifest_org, target_sat):
+    """Attempt to hit the /available_repositories endpoint with no failures
+
+    :id: f4c9d4a0-9a82-4f06-b772-b1f7e3f45e7d
+
+    :Steps:
+        1. Enable a Red Hat Repository
+        2. Attempt to hit the enpoint:
+           GET /katello/api/repository_sets/:id/available_repositories
+        3. Verify Actions::Katello::RepositorySet::ScanCdn task is run
+        4. Verify there are no failures when scanning for repository
+
+
+    :expectedresults: Actions::Katello::RepositorySet::ScanCdn task should succeed and
+        not fail when scanning for repositories
+
+    :customerscenario: true
+
+    :BZ: 2030445
+    """
+    rh_repo_id = target_sat.api_factory.enable_rhrepo_and_fetchid(
+        basearch=constants.DEFAULT_ARCHITECTURE,
+        org_id=module_sca_manifest_org.id,
+        product=constants.PRDS['rhel8'],
+        repo=constants.REPOS['rhst8']['name'],
+        reposet=constants.REPOSET['rhst8'],
```

Comment:
I would prefer single source of truth:
```suggestion
        product=constants.REPOS['rhst8']['product'],
        repo=constants.REPOS['rhst8']['name'],
        reposet=constants.REPOS['rhst8']['reposet'],
```
This way you can have `'rhst8'` as a parameter, not hard-coded, and possibly parametrize the test with different repos.

Last thing, I would rather use more up-to-date repo than `satellite-tools-6.9-for-rhel-8-x86_64-rpms`, or some more frequent one(s). The BZ talks about `rhel-7-server-extras-rpms`, which would be `REPOS['rhel7_extra']`.